.env

```diff
@@ -1,2 +1,2 @@
-VUE_APP_WALLET_URL=https://wallet.superhero.com
-VUE_APP_JITSI_URL=test.league.aeternity.org
+VUE_APP_WALLET_URL=https://develop.wallet.z52da5wt.xyz
```

Comment:
I think we should release the wallet firstly

Reply:
https://github.com/aeternity/superhero-ui/pull/745